util.py

```diff
@@ -437,3 +437,68 @@ def getPlainURL(self):
             assert self.password is None
 
             return self.url
+
+def readCmdlineMounts():
+    class Mount:
```

Comment:
Would it be better to extract this class such that it's not contained within a method, it might be useful elsewhere?

Reply:
I'm a bit divided on just fix this issue and do it completely and properly. In that case supporting all cases and maybe do everything from python instead of splitting between here and an external script